products/silverstripe.md

```diff
@@ -74,6 +74,41 @@ releases:
     latest: "4.10.11"
     latestReleaseDate: 2022-11-21
 
+-   releaseCycle: "3.7"
+    releaseDate: 2018-06-08
+    support: true
+    eol: 2021-09-30
+    latest: "3.7.7"
+    latestReleaseDate: 2021-06-02
+
+-   releaseCycle: "3.1"
+    releaseDate: 2013-10-01
+    support: true
+    eol: 2016-12-31
```

Comment:
Where can I confirm this date ?

Reply:
I actually don't know. Following the old release policy, it should have gone eol when 3.3 was released (https://github.com/silverstripe/silverstripe-framework/releases/tag/3.3.0), but my work notes say otherwise. because 3.1 was the last version not adhering to semantic versioning support was likely extended. But I'll have to find a reference to that.

Reply:
here we go: https://www.silverstripe.org/blog/support-timeline-update-where-are-we-heading/